integreat_cms/static/src/js/media-management/component/directory-content.tsx

```diff
@@ -34,37 +34,38 @@ const DirectoryContent = ({
 }: Props) => {
     // The file index contains the index of the file which is currently opened in the sidebar
     const [fileIndex, setFileIndex] = fileIndexState;
-
     return (
         <div className="grid grid-cols-gallery max-h-full gap-1">
-            {mediaLibraryContent.map((entry: MediaLibraryEntry, index: number) =>
-                entry.type === "directory" ? (
-                    <Link key={entry.id} href={`/${entry.id}/`} media-library-link>
-                        <DirectoryEntry
-                            directory={entry as Directory}
+            {mediaLibraryContent.map(
+                (entry: MediaLibraryEntry, index: number) =>
+                    (globalEdit || !entry.isHidden) &&
+                    (entry.type === "directory" ? (
```

Comment:
```suggestion
                    entry.type === "directory" ? (
```
Now, we can remove this condition again, since the objects are already filtered on the server side.